cmd/sink/filters/filters.go

```diff
@@ -283,7 +289,7 @@ func (f *Filters) handleFsEvent(event fsnotify.Event) {
 			slog.Info("Could not read global filters", "err", err)
 			break
 		}
-		if fileName == globalKey {
+		if fileName == getGlobalKey() {
```

Comment:
Why not call this once, instead of here and 30 lines down?

Reply:
done